CMakeLists.txt

```diff
@@ -389,12 +389,11 @@ endif(QUDA_VERBOSE_BUILD)
 
 
 set(CUDA_NVCC_FLAGS_DEVEL ${QUDA_NVCC_FLAGS} -Xcompiler -Wno-unknown-pragmas -O3 -lineinfo CACHE STRING
-    "Flags used by the CUDA compiler during regular development builds."
-    FORCE )
```

Comment:
removing `FORCE` here results in empty CUDA_NVCC_FLAGS_DEVEL for me and this breaks compilation. Not sure whether this is specific to my setup.